website/static/js/pages/profile-settings-addons-page.js

```diff
@@ -89,6 +91,18 @@ for (var i=0; i < addonEnabledSettings.length; i++) {
    }
 }
 
+/* Before closing the page, Check whether the newly checked addon are updated or not */
+$(window).on('beforeunload',function() {
+    //new checked items but not updated
+    var checked = uncheckedOnLoad.filter($('#selectAddonsForm input:checked'));
```

Comment:
jQuery's `.filter()` should allow you to pass in a selector directly, making the `$()` unnecessary.

```javascript
var checked = uncheckedOnLoad.filter($('#selectAddonsForm input:checked'));
```